### DISCUSSION TITLE
Cards: Effects performed by a explicit superpower

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#21] [Ideas]
These are cards I've interpreted to have Event effects which is explicitly performed (in whole or partially) by a certain superpower. This might be an incorrect reading of the original text, thus we need to make inventory of them and correct them where incorrectly interpreted.

I complied this list somewhat quickly, so please look through the cards in cards.mnt (it's a plain text file) to find anything I've missed, including cards that *should* be explicitly performed by someone but I've misunderstood as such. Also please look through the card texts in general, a final check before we call these text updates done. For this purpose I will now close the two PRs as we've processed all feedback, I will update cards.mnt, and I will remove changes.md as it has fullfilled its purpose. All this in a *force* push.

**Probably already correct but included because I just did a ctrl+f**

Party Congress
> China adds 1 Influence .. *or* removes 1 Chinese Influence ..

Rupert Murdoch
> US vie for Supremacy ..

Russian oligarchs
> Choose an Economy Slot. Russia gains direct Supremacy ..

Hu Jintao
> China chooses two countries ..

New START
> US and Russia may elect ..

Osama bin Laden
> .. US removes 1 Influence ..
> .. US .. removes 1 US Influence ..

**Possibly incorrect**

Wolfowitz doctrine
> US removes 2 East Influence ..

Austerity plans
> US adds 1 US Influence ..
> EU adds 1 EU Influence ..

Nord Stream 1
> Russia may add 1 Russian Influence ..

PRISM
> US removes 1 US Influence ..

Madrid bombings
> **EU player chooses**: EU removes 1 West Influence ..

**Comment by rodrigoSantamaria** [comment]
All the ones you cite here are correct. 
These are other cards where the text is executed by a specific superpower:
4 -Boris Yeltsin
9- EFTA Agreement
12- Empire of War
...

I don't know if go on, maybe these ones were already checked and nothing wrong in wording was found by you?
Ah, I guess you mean only if there's a decision to be made?

In that case, these are also on board:
28 - Northern Adhesion
22- Maastrich Treaty
23- Made in China
47- 9/11 Attacks
68 - Hu Jintao
78 - Nord Stream 1
80 - PRISM



Cheers,
Rodrigo

**Reply by RebelusQuo** [reply]
I did not list cards with sentences starting with the phrase "[power] [gains/loses] N Influence" because wherever their instruction targeted an exact target set I simply interpreted it as "[add/remove] N [power] Influence", but giving it a second thought we did establish that "N Influence" means "Influence for any power, according to the player's wish". Therefor for some of these perhaps it should be "[power] [adds/removes] N Influence", similar to the original? So here are those cards, as far as I could Ctrl+F:

4 Boris Yeltsin, original:
> US gains 1 influence in Russia.

Interpreted as:
> Add 1 US Influence in Russia.

9 EFTA agreement, original:
> EU gains 1 influence in Norway. ..

Interpreted as:
> Add 1 EU Influence in Norway.

15 FSB creation, original:
> Russia gains 1 influence in Russia. ..

Interpreted as:
> Add 1 Russian Influence in Russia.

41 South Lebanon Conflict, original:
> US gains 1 influence in Lebanon. ..

Interpreted as:
> Add 1 US Influence in Lebanon.

45 Wolfowitz doctrine, correct as previously discussed.

71 Invasion of Crimea, original:
> Russia gains 1 influence in Ukraine, ..

Interpreted as:
> Add 1 Russian Influence in Ukraine.

80 PRISM, correct as previously discussed.

83 Syrian civil war, original:
> .. Russia gains 1 influence in Syria and ..

Interpreted as:
> .. , then add 1 Russian Influence.

86 Viktor Yushchenko, original:
> Russia .. and 1 influence in Ukraine.

Interpreted as:
> Remove 1 Russian Influence from Ukraine.

87 Vladimir Putin, original:
> Russia gains 1 (2 if FSB is in play) influence in Russia. ..

Interpreted as:
> Add 1 Russian Influence in Russia.
> Add an additional Influence if the 'FSB creation' effect is active.

So in short, for which of these cards was it correct to assume the Influence is always for the mentioned superpower, and for which are the superpower able to choose among any Influence?

**Reply by RebelusQuo** [reply]
When it comes to the additional cards:

22 Maastrich Treaty, original had the phrase "Add 1 EU influence" so should this also be a choice for EU? like:
> EU adds 1 EU Influence in each ..

23 Made in China, original had the phrase "Add 1 China influence" so should this also be a choice for China? like:
> China adds 1 Chinese Influence in each ..

28 Northern adhesion, original did have the phrase "EU adds ..", so I assume this should be EU's choice of countries.
> EU adds 1 EU Influence in one country among Austria, Finland, and Sweden.

47- 9/11 Attacks, original had the phrase ".. gain 2 US influence in US and 1 in any two EU countries." so should this also be a choice for US, and is the Influence in EU countries also strictly US Influence? like:
> US adds 2 US Influence in the United States and 1 US Influence in each of two EU countries.

68 - Hu Jintao, previously discussed but only the first sentence. On BGG you had answered another person that the other countries where the Influence are added are decided by the performing player, which may be Russia. Correct?

78 - Nord Stream 1, correct as previously discussed. However, I spot a "may" that I thought I had removed. should be:
> Russia adds 1 Russian Influence in one EU country, except UK.

80 - PRISM, correct as previously discussed.

**Reply by RebelusQuo** [reply]
Long lists of small details, but we've almost reached port now, so to speak.

**Reply by rodrigoSantamaria** [reply]
Every interpretation is correct. Sorry to make you go on these small details.

**Reply by RebelusQuo** [reply]
Pardon me, I should have been more explicit. In my first reply what I wrote were my interpretations are what you *currently* find in cards.mnt. However, in my second reply the quotes are suggestions of what you might have intended. Do you mean to verify my first reply, and tell me the new suggestions in the second reply are the correct ones?

**Reply by rodrigoSantamaria** [reply]
Exactly, both things :)

**Comment by RebelusQuo** [comment]
I've force pushed again, hopefully we've covered everything now. Please have a look once you got the time. How about I continue with the rules document next?

**Comment by rodrigoSantamaria** [comment]
Yes, the rules are the next place to go. I'll do a full review of the cards this weekend.

**Reply by RebelusQuo** [reply]
Perfect, and close.
